helix-term/Cargo.toml

```diff
@@ -76,3 +76,5 @@ helix-loader = { version = "0.6", path = "../helix-loader" }
 smallvec = "1.10"
 indoc = "2.0.0"
 tempfile = "3.4.0"
+pretty_assertions = "1.3.0"
+build-fs-tree = "0.4.1"
```

Comment:
Was this only used for debugging? I see new dependencies but no new tests. Should remove this before merge

Reply:
Either way I'd like us to drop these additional dependencies, they're only used in tests but I'm not convinced they're worth the extra crates

Reply:
Resolved via https://github.com/pinelang/helix-tree-explorer/commit/33542e9ddbd5776727531eb19d2eda83449a592e